src/lib.rs

```diff
@@ -885,18 +910,55 @@ pub enum ObjectProperty {
     Pattern(i32),
     PatternTiling(Vec2),
     PatternOffset(Vec2),
+    Bounce(bool),
+    RestoreVelocity(bool),
     Sprite(String),
     Trigger(bool),
     Health(f32),
     DamageFromJump(bool),
     DamageFromDash(bool),
     ReverseDirOnDamage(bool),
     Floating(bool),
+    LinkedObjects(Vec::<i32>),
     FlipX(bool),
     FlipY(bool),
     Text(String),
     FontSize(f32),
     EditorColour(Colour),
+    Colour2(Colour),
+    Colour3(Colour),
+    Colour4(Colour),
```

Comment:
What exactly are these?

Reply:
i think in general all the variants in this enum should have much better names and docs etc, but that's a thing for later :)

Reply:
That's how they're called in game as well 😭 (and you can see them when you for example want to recolour a slingshot

Reply:
oh yea fair enough :3